administrator/components/com_contact/src/Model/ContactModel.php

```diff
@@ -297,6 +305,23 @@ public function save($data)
 			$createCategory = !CategoriesHelper::validateCategoryId($data['catid'], 'com_contact');
 		}
 
+		if (isset($data['image']))
+		{
+			// Initial version
+			$initImage = explode("#", $app->getUserState("com_contact.image"))[0];
+
+			// Final version
+			$finalImage = explode('#', $data['image'])[0];
+
+			MediaHelper::generateResponsiveFormImages($initImage, $finalImage);
```

Comment:
It's sad, that you need to write so much (nearly) duplicated code. But I don't have a solution yet :/